lib/Agrammon/OutputFormatter/JSON.pm6

```diff
@@ -6,36 +6,38 @@ use Agrammon::Outputs::FilterGroupCollection;
 sub output-as-json(
     Agrammon::Model $model,
     Agrammon::Outputs $outputs,
-    $language, $prints,
+    $language,
+    @print-set,
     Bool $include-filters,
     Bool :$all-filters = False
 ) is export {
-    return get-data($model, $outputs, $include-filters, $language, $prints);
+    return get-data($model, $outputs, $include-filters, @print-set, $language);
 }
 
 sub output-for-gui(Agrammon::Model $model,
                    Agrammon::Outputs $outputs,
                    :$include-filters,
                    :$language
                    ) is export {
+    my @print-set = ('All');
```

Comment:
Unimportant, but don't need the parens here in Raku (and in a few other places in this PR).

Reply:
Removed